### ISSUE TITLE
Banghazi should be Benghazi

### ISSUE THREAD

**Issue by almccon** [nvkelso/natural-earth-vector#471]
The city of Benghazi has the incorrect (or at least extremely uncommon) spelling of "Banghazi" as NAME, NAMEASCII, MEGANAME, LS_NAME, and GN_ASCII. It has the correct "Benghazi" spelling for NAMEALT, ADM1NAME, CITYALT, and name_en.

The Wikidata item doesn't show the "Banghazi" spelling at all. https://www.wikidata.org/wiki/Q40816

**Comment by nvkelso** [comment]
Uff, good find. @mizmay can you add this one to your PR, too, please?

**Comment by ImreSamu** [comment]
side notes:

> The city of Benghazi has the incorrect (or at least extremely uncommon) spelling of "Banghazi

probably based on the arabic-latin transliteration  ( "Banġāzī" )  + unaccented version 

* https://en.wikipedia.org/wiki/Benghazi#cite_note-14
" Arabic: بنغازي‎[6] **Banġāzī,** Libyan Arabic: [bənˈʁɑːzi] (About this soundlisten); Italian: Bengasi; Turkish: Bingazi; Berber languages: Bernîk; also: Bengasi,[7] Benghasi,[8] Banghāzī,[9] Binghāzī,[10] Bengazi;[11] Ancient Greek: Βερενίκη (Berenice)[12] and Hesperides.[13]"

* https://geographic.org/geographic_names/name.php?uni=-4461835&fid=3757&c=libya
   * "Full Name with QWERTY characters (see definition): **Banghazi**"

> The Wikidata item doesn't show the "Banghazi" spelling at all. https://www.wikidata.org/wiki/Q40816

I have added the  "Banġāzī"  to the english wikidata("Also known as")  based on the en-wiki page.
( imho:  this is important for any wikidata<--->geodata matching algorithm )

**Comment by mizmay** [comment]
Oops I missed this issue until now. Skimming through original issue and comments, proposed fix:

field | old | new
--- | --- | ---
NAME | Banghazi | Benghazi
NAMEALT | Benghazi | Banġāzī
NAMEPAR | | 
NAMEASCII | Banghazi | Benghazi
ADM1NAME | Benghazi | Benghazi 
MEGANAME | Banghazi | Benghazi                                   
LS_NAME | Banghazi | Benghazi

& verified that the wikidataid is correct and will pick up the localizations appropriately.

```
ne_10m_populated_places %>% filter(wikidataid == 'Q40816') %>% select(contains('NAME', ignore.case = T)) %>% as.data.frame() %>% t()
           [,1]                                            
NAME       "Banghazi"                                      
NAMEPAR    "NA"                                            
NAMEALT    "Benghazi"                                      
NAMEASCII  "Banghazi"                                      
SOV0NAME   "Libya"                                         
ADM0NAME   "Libya"                                         
ADM1NAME   "Benghazi"                                      
NAMEDIFF   0                                               
GEONAMEID  88319                                           
MEGANAME   "Banghazi"                                      
LS_NAME    "Banghazi"                                      
GEONAMESNO "Geonames ascii name + lat.d + long.d matching."
name_en    "Benghazi"                                      
name_de    "Bengasi"                                       
name_es    "Bengasi"                                       
name_fr    "Benghazi"                                      
name_pt    "Bengasi"                                       
name_ru    "Бенгази"                                       
name_zh    "班加西"                                        
name_ar    "NA"                                            
name_bn    "NA"                                            
name_el    "NA"                                            
name_hi    "NA"                                            
name_hu    "NA"                                            
name_id    "NA"                                            
name_it    "NA"                                            
name_ja    "NA"                                            
name_ko    "NA"                                            
name_nl    "NA"                                            
name_pl    "NA"                                            
name_sv    "NA"                                            
name_tr    "NA"                                            
name_vi    "NA"
```

**Comment by mizmay** [comment]
Late edition but this issue should be resolved with 4097952dfff9ada2faca3b2849da4cf2d9c0dcb4
